Patient view

emilyville opened the pull request:
fixes #296, 
fixes #277 , 
fixes #260  
Clean up / remove some old scss files from pre-emily
change title of primary scss file to "global" 
change load order of scss to be better
incorporate (some of) fatima's suggestions
side panel better
nav tweeks
made labels more consistent

make submit buttons consistent across forms
make a new class for time-span instrument (i.e START, 90 days) titles for instruments to make cleaner
remove extra boxes for instruments

Commit:
fixes #296, change sass compile order, incorporates some of Fatima suggestions into patient nav, and make patient side control panel closer to complete

Commit:
fixes #277

Commit:
fixes #260

Commit:
fix the tests

Commit:
make submit buttons consistent and remove some rampant double div action

emilyville commented:
Tests are passing for me locally, confused about why they are failing here.

awong-dev commented on spec/features/episodes_of_care/omrs_spec.rb:
```diff
@@ -23,7 +23,7 @@
     expect(page).to have_content patient1.ssn
 
-    click_link("OMR")
+    click_link("+ OMR"))
```
extra right paren.

awong-dev commented:
@emilyville https://travis-ci.org/department-of-veterans-affairs/SCIInterim/builds/77597765#L620 shows the error. Uncertain how your tests could be passing locally...

awong-dev commented:
Does this actually fix #286? If not, please update comment so we don't close the issue on merge.

emilyville commented:
@awong-dev good catch. the commit was right (277) and the comment was wrong!

Commit:
remove extra paren

emilyville commented:
Ok, all issues fixed aside from tests, which have a routing error - travis says only for Acute Rehab and Annual Eval, but I am _also_ get it locally for OMR.

It's saying "No route matches [PATCH] "/patients/2/acute_rehabs". I can't think of any reason my changes would have caused this.

awong-dev commented on app/views/layouts/patient_edit.html.slim:
```diff
@@ -8,56 +8,64 @@
+                    | completion date: 
+                    / TODO: Make sure this always exists
+                    / = episode.eval_completed
+          - if @patient.episode_of_cares.size < 1 
```
Use `@patient.episode_of_cares.empty?` instead to be more idiomatic.

awong-dev commented on app/views/shared/_asia_fields.html.slim:
```diff
@@ -35,22 +34,26 @@
                   label: false,
                   collection: Asia.collections[:neurological_motor_level_right]
       .col-md-4
+        = wrapped_form_for(@patient) do |f|
```
This is what's causing the form submit to break.  wraped_form_for creates another <form> element within the embedding <form> and confuses all the later code.

If we have to do this, we should pass in the parent form builder as another variable other than |f|, however I'm a bit confused at the UI here.  Should the `sci_type` be per patient, or per Asia evaluation? If it's per patient, having it on the asia widget is confusing...

awong-dev committed:
Remove sci_type and use empty? idiom.

The inclusion of the patient.sci_type field into the _asia_fields
partial by using wrapped_form_for was causing a nested form attribute
that confused simple_forms breaking the sumbit button. It's unclear what
the right UI paradigm is for this field so moving back for now. This
reopens #260.
